app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    """
+    this function designed for copying two files
+    :param command: "cp file_to_copy.format new_file.format"
+    """
+    file_1 = command.split(" ")[1]
+    file_2 = command.split(" ")[2]
```

Comment:
It can be done simpler
```suggestion
    cmd, file_1, file_2 = command.split()

```

Reply:
Wow! 
very interesting way!
Thank you!

Reply:
It's not the functionality of a `split()` function. It only returns your list and after you assign the value of the list to variables.
Something like this 
```
list = command.split() # the value list will be [cmd, file1, file2]
cmd, file_1, file_2 = list
```
 So it's more about work with list